wire-ios-sync-engine/Tests/Source/Use cases/GetE2eIdentityCertificatesUseCaseTests.swift

```diff
@@ -63,22 +63,23 @@ final class GetE2eIdentityCertificatesUseCaseTests: XCTestCase {
     ) -> WireCoreCrypto.WireIdentity {
         return .init(
             clientId: clientID,
-            handle: handle,
+            handle: "wireapp://%40\(handle)",
```

Comment:
question: Shouldn't this be `wireapp://%40\(handle)@\(domain)`?

Reply:
The handle being passed in is already "\(handle)@\(domain)"